From 396245460a90ce8be4d3af1cc32516593d5f4d96 Mon Sep 17 00:00:00 2001
From: Robin Gong <yibin.gong@nxp.com>
Date: Fri, 17 Feb 2017 15:59:25 +0800
Subject: [PATCH 1530/5242] MLK-13994-2 ARM: dts: imx7ulp: add heartbeat
 device node

commit  7a5b84e27cbac46c9b51cb42d83600f0071709dd from
https://source.codeaurora.org/external/imx/linux-imx.git

add heartbeat device node to add suspend/resume in
arch/arm/mach-imx/pm-rpmsg.c

Signed-off-by: Robin Gong <yibin.gong@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7ulp.dtsi |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/imx7ulp.dtsi b/arch/arm/boot/dts/imx7ulp.dtsi
index dc7e8ef..c553c47 100644
--- a/arch/arm/boot/dts/imx7ulp.dtsi
+++ b/arch/arm/boot/dts/imx7ulp.dtsi
@@ -597,6 +597,10 @@
 		};
 	};
 
+	heartbeat-rpmsg {
+		compatible = "fsl,heartbeat-rpmsg";
+	};
+
   imx_ion {
     compatible = "fsl,mxc-ion";
     fsl,heap-id = <0>;
-- 
1.7.9.5

